pkg/scanners/scannerv4/convert_test.go

```diff
@@ -295,6 +295,63 @@ func TestComponents(t *testing.T) {
 	}
 }
 
+func TestSetEPSS(t *testing.T) {
+	testcases := []struct {
+		name       string
+		epssDetail *v4.VulnerabilityReport_Vulnerability_EPSS
+		expected   *storage.EmbeddedVulnerability
+		wantErr    bool
+	}{
+		{
```

Comment:
add following  test cases 
1. when epss is null in vulnerability report
2. when one of probability or percentile is missing